action.yml

```diff
@@ -21,7 +21,7 @@ inputs:
     description: 'Additional command line options'
     required: false
   issue_title:
-    description: 'The title for the GitHub issue that is created'
+    description: 'The title for the GitHub issue to be created'
```

Comment:
If we change here we should change in the README too.

Reply:
added in the readme